BSP/STM32F413/Src/BSP_WDTimer.c

```diff
@@ -4,16 +4,17 @@
 #include "stm32f4xx.h"
 
 /**
- * @brief   Initialize the watch dog timer.
- * @param   None
+ * @brief   Initialize the watch dog timer. If in fault state, set watchdog reload to max value so we don't trip for watchdog in fault state again.
+ * @param   fault: true if in fault state, false otherwise
  * @return  None
  */
-void BSP_WDTimer_Init(void) {
+void BSP_WDTimer_Init(bool fault) {
     // Independent Watchdog Init
-    // IWDG has a timeout value of 4.096 / 8 seconds (about 500 ms).
+    // IWDG has a timeout value of 4.096 / 16 seconds (about 250 ms).
```

Comment:
was the previous comment wrong? when dividing by 4 but it said 4.096 / 8s? just wanna double check

Reply:
yes it was